src/Discovery/Utils/MDSUtils/MDSUtils.jsx

```diff
@@ -43,4 +43,29 @@ const loadStudiesFromMDS = async (guidType = 'discovery_metadata') => {
   }
 };
 
-export default loadStudiesFromMDS;
+export const getSomeStudiesFromMDS = async (guidType = 'discovery_metadata', numberOfStudies = 10) => {
```

Comment:
I don't think we should re-write this function since a good chunk of its logic already exists in `loadStudiesFromMDS()` above
we should be able to make `loadStudiesFromMDS()` to take `LIMIT` as an argument, something like
```
export const loadStudiesFromMDS = async (guidType = 'discovery_metadata', limit=2000)
```
and if we need partial loading, just call this function with a smaller `limit` value

Reply:
Good idea! Per our discussion, the loadStudiesFromMDS() has been refactored to take in a numeric `fetchSize` as a parameter and a boolean `loadAllMetadata`, both with default assignments. 

When called with `loadAllMetadata === false` the while loop exits after a single fetch. The calls to `getSomeStudiesFromMDS` have been replaced to calls with `loadAllMetadata` with additional parameters. The unit tests for MDSUtils have been updated accordingly.